queries.md

```diff
@@ -16,47 +16,165 @@ SELECT \* FROM users;
 Using the `movies_db` database, write the correct SQL queries for each of these tasks:
 
 1.  The title of every movie.
+```
+SELECT title 
+    FROM movies;
+```
 
 2.  All information on the G-rated movies.
+```
+SELECT * 
+    FROM movies 
+    WHERE rating = 'G';
+```
 
 3.  The title and release year of every movie, ordered with the
     oldest movie first.
+```
+SELECT title, release_year 
+    FROM movies 
+    ORDER BY release_year;
```

Comment:
best to be explicit. default order can be different among different DB frameworks
```suggestion
    ORDER BY release_year ASC;
```